deploy/victoriametrics/crs/vmagent_rbac.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: vmagent
+  namespace: monitoring-system
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: vmagent
+rules:
+  - apiGroups: ["","networking.k8s.io","extensions"]
+    resources:
+      - nodes
+      - nodes/metrics
+      - services
+      - endpoints
+      - endpointslices
+      - pods
+      - app
+      - ingresses
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources:
+      - namespaces
+      - configmaps
+    verbs: ["get"]
+  - nonResourceURLs: ["/metrics","/metrics/resources"]
+    verbs: ["get"]
+  - apiGroups:
+      - route.openshift.io
+      - image.openshift.io
```

Comment:
why is it openshift.io?

Reply:
copy pasted from victoriametrics operator documentation.